test/WebJobs.Script.Tests.Integration/ContainerManagement/LinuxContainerInitializationHostServiceTests.cs

```diff
@@ -79,12 +79,12 @@ public async Task Assigns_Context_From_CONTAINER_START_CONTEXT()
             _environment.SetEnvironmentVariable(ContainerEncryptionKey, containerEncryptionKey);
             AddLinuxConsumptionSettings(_environment);
 
-            _instanceManagerMock.Setup(manager => manager.StartAssignment(It.Is<HostAssignmentContext>(context => hostAssignmentContext.Equals(context)), It.Is<bool>(w => !w))).Returns(true);
```

Comment:
rather than remove these checks, can you move them into the preceeding It.Is check to verify !context.IsWarmupRequest?